src/main/java/org/stellar/sdk/responses/operations/ExtendFootprintTTLOperationResponse.java

```diff
@@ -3,7 +3,9 @@
 import com.google.gson.annotations.SerializedName;
 
 /**
- * Represents BumpFootprintExpiration operation response.
+ * Represents ExtendFootprintTTL operation response.
```

Comment:
Currently, the code for Horizon has not been updated, so we need to update this link later.